packages/terra-form-select/tests/wdio/select-spec.js

```diff
@@ -719,6 +719,22 @@ Terra.describeViewports('Select', ['tiny'], () => {
       });
 
       Terra.it.validatesElement('selected-option');
+
+      after(() => {
+        browser.refresh(); // remove selected option
+      });
+    });
+
+    describe('combobox should open close and open dropdown by clicking root and hover first option', () => {
```

Comment:
nit: should hover on first option on reopen after closed by clicking on root. 
OR 
Something more appropriate

Reply:
updated here b89c45958fa2c8099a9dd95b3e37a52e61c74105